### PR TITLE
Add instructions for manually building the plugin

### PR DESCRIPTION
**Pull request by etanshaul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,17 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
...
-(coming soon)
+If you'd like to try out the plugin pre-release, you can build it from source and install it into your IDE:
+
+1) Clone this repository to your machine.
+2) Build the plugin from the root of the repository:<br />
+   `./gradlew buildPlugin`
+3) Find the `.zip` binary in the root of the repository under: <br />
+   `build/distributions/google-container-tools-intellij.zip`
+4) Install the plugin into your JetBrains IDE:
+   1) Go to `Preferenes > Plugins`
```
Comment (ivanporty) [inline README.md]: Preferences missing `c` it seems

File: README.md [hunk 2]
```diff
@@ -27,7 +27,17 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
 
 ## Installing the plugin into your IDE
 
-(coming soon)
+If you'd like to try out the plugin pre-release, you can build it from source and install it into your IDE:
+
+1) Clone this repository to your machine.
```
Comment (ivanporty) [inline README.md]: Insert `git clone ...` string here just for easy copy-paste step-by-step instructions?

File: README.md [hunk 3]
```diff
@@ -27,7 +27,18 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
...
+If you'd like to try out the plugin pre-release, you can build it from source and install it into your IDE:
+
+1) Clone this repository to your machine:<br />
+   `git clone git@github.com:GoogleContainerTools/google-container-tools-intellij.git`
+2) Build the plugin from the root of the repository:<br />
+   `./gradlew buildPlugin`
+3) Find the `.zip` binary in the root of the repository under: <br />
+   `build/distributions/google-container-tools-intellij.zip`
+4) Install the plugin into your JetBrains IDE:
+   1) Go to `Preferences > Plugins`
```
Comment (ivanporty) [inline README.md]: Mention that this menu is named `Settings` on non-Mac OS, like Linux?

File: README.md (hunk 1 above)
Comment (etanshaul) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (etanshaul) [reply README.md]: added

File: README.md (hunk 3 above)
Comment (etanshaul) [reply README.md]: I'm not on my linux machine now: is it called Settings instead of Preferences in that case?

File: README.md [hunk 4]
```diff
@@ -27,7 +27,18 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
...
+   2) Select the settings gear icon and then `Install Plugin From Disk`
```
Comment (ivanporty) [inline README.md]: What do you think about adding a single screenshot here where this menu is shown opened? It is kind of hidden in newer versions of IDEs and some users may not be familiar enough with this feature.

File: README.md (hunk 3 above)
Comment (ivanporty) [reply README.md]: It is, yes.

**Comment by etanshaul** [comment]
I think I addressed everything. PTAL

File: README.md [hunk 5]
```diff
@@ -27,7 +27,19 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
...
+1) Clone this repository to your machine:<br />
+   `git clone git@github.com:GoogleContainerTools/google-container-tools-intellij.git`
+2) Build the plugin from the root of the repository:<br />
+   `./gradlew buildPlugin`
+3) Find the `.zip` binary in the root of the repository under: <br />
+   `build/distributions/google-container-tools-intellij.zip`
+4) Install the plugin into your JetBrains IDE:
+   1) Go to `Preferences/Settings > Plugins`
+   2) Select the settings gear icon and then `Install Plugin From Disk`:
+      ![manual-install](src/main/resources/images/plugin-manual-install.png =500x)
```
Comment (ivanporty) [inline README.md]: Should we put the documentation images into sources? What if we put them separately, maybe in `docs/images` or something like that? It feels we usually put something used by code into `resources`.

**Pull request merged**